### PR TITLE
chore: update anchor href definition

### PR DESCRIPTION
**Pull request by YiniXu9506**
1. update anchor href
2. fix some lint errors

### CODE REVIEW

File: 2016-11-09-Deep-Dive-into-TiKV.md [hunk 1]
```diff
@@ -7,19 +7,17 @@ aliases: ['/blog/2016/11/09/Deep-Dive-into-TiKV/', '/blog/2016/11/09/deep-dive-i
...
 - [Architecture](#architecture)
 - [Protocol](#protocol)
 - [Raft](#raft)
-- [Placement Driver (PD)](#placement-driver)
+- [Placement Driver (PD)](#placement-driver-pd)
 - [Transaction](#transaction)
 - [Coprocessor](#coprocessor)
 - [Key processes analysis](#key-processes-analysis)
-  - [Key-Value operation](#key-value-operation)
+  - [Key-Value operation](#keyvalue-operation)
```
Comment (CaitinChen) [inline 2016-11-09-Deep-Dive-into-TiKV.md:20]:
```suggestion
  - [Key-Value operation](#key-value-operation)
```

File: 2017-05-23-perconalive17.md [hunk 2]
```diff
@@ -10,27 +10,25 @@ categories: ['MySQL Scalability']
...
-- [ACID Transaction](#transaction)
-- [Distributed SQL](#distsql)
-- [TiDB SQL Layer Overview](#sql)
-- [What Happens behind a query](#query)
-- [Distributed Join (HashJoin)](#hashjoin)
-- [Tools Matter](#tools)
-- [Use Cases](#usecase)
+- [Speaker introduction](#speaker-introduction)
+- [What would you do when…](#what-would-you-do-when)
+- [TiDB Project - Goal](#tidb-project---Goal)
```
Comment (CaitinChen) [inline 2017-05-23-perconalive17.md:17]:
```suggestion
- [TiDB Project - Goal](#tidb-project---goal)
```

File: 2017-07-11-tidbinternal2.md [hunk 3]
```diff
@@ -8,23 +8,21 @@ aliases: ['/blog/2017/07/11/tidbinternal2/']
...
 
 ## Table of Content
 
-+ [Mapping the Relational Model to the Key-Value Model](#map)
-+ [Metadata Management](#meta)
-+ [Architecture of SQL on Key-Value](#sqlonkv)
-+ [SQL Computing](#sqlcompute)
-+ [Distributed SQL Operation](#dissql)
-+ [Architecture of the SQL Layer](#sqlarch)
++ [Mapping the Relational Model to the Key-Value Model](#mapping-the-relational-model-to-the-keyvalue-model)
```
Comment (CaitinChen) [inline 2017-07-11-tidbinternal2.md:15]:
```suggestion
+ [Mapping the Relational Model to the Key-Value Model](#mapping-the-relational-model-to-the-key-value-model)
```

File: 2016-11-09-Deep-Dive-into-TiKV.md (hunk 1 above)
Comment (YiniXu9506) [reply 2016-11-09-Deep-Dive-into-TiKV.md:20]: `#key-value-operation` is correct

**Review by CaitinChen — APPROVED** [review]
LGTM

**Pull request merged**